### ISSUE TITLE
Hacer el gráfico "embebible"

### ISSUE THREAD

**Issue by electroduende21** [numeroteca/contratosmenores#13]
El CM del OCM de Paracuellos me ha comentado que sería muy bueno si el gráfico con los contratos menores pudiéra ser incrustable en su blog.
Ganaríamos mucho si existiera la opción de que sean "embebibles"

**Comment by numeroteca** [comment]
Creo que debería ser una versión reducida del gráfico, sin la parte del menú y quitando casi todo de la parte de abajo ¿cómo os imaginais que fuera?

**Comment by electroduende21** [comment]
Sería estático entonces?
Sería una versión reducida del gráfico ya filtrada por un sólo proveedor?¿?

**Comment by numeroteca** [comment]
No, sería interactivo igualmente. Embeber una web en otra ya se puede hacer
con "iframe" pero tendrías todas las pestañas de acceso a las otras
visualizaciones, el título, footer... lo que decía es ¿qué cosas quereis
que incluya estar versión embebible?

**Comment by numeroteca** [comment]
Un ejemplo de los modos de visualizar:

+ visualización en la web: http://pageonex.com/numeroteca/corrupcion-espana-septiembre-2016/
+ visualizació nembebido en otra web http://pageonex.com/numeroteca/corrupcion-espana-septiembre-2016/embed

¿se entiende?

**Comment by electroduende21** [comment]
Yo añadiría los botones con los centros de gasto, y 
una selección de los otros botones. (los 5 top suministradores?)
Los botones para agrupaciones de descripción (thinglist), creo que falta pulirlos, pero se podrían poner.

**Comment by numeroteca** [comment]
Haced la prueba.
Con este código podeis meter el iframe:

`<div id='wrap' style='width: 100%; height: 800px; padding: 0; overflow: hidden;'><iframe style='width: 1280px; height: 1100px; -ms-zoom: 0.75; -transform: scale(0.68);-moz-transform: scale(0.75);-moz-transform-origin: 0 0; -o-transform: scale(0.75);-o-transform-origin: 0 0; -webkit-transform: scale(0.75);-webkit-transform-origin: 0 0;max-width: none; overflow: hidden; border: 1px solid white;' src='http://numeroteca.org/contratosmenores/paracuellos-2014_embed.html'></iframe></div>`

Modificando width, height y todos los nímeles de zoom (ahora todos fijados a 75%) se puede cambiar el tamaño. Es cuestión de ir probando qué encaja mejor.

La url a embeber es http://numeroteca.org/contratosmenores/paracuellos-2014_embed.html

**Comment by numeroteca** [comment]
Aquí la nueva URL a embeber: https://lab.montera34.com/contratosmenores/zaragoza/facturas/2017/index_embed.html

Haría más cortos los nombres de los centros para que fuera todo más compacto. Probad a embeberlo.

`<div id='wrap' style='width: 100%; height: 800px; padding: 0; overflow: hidden;'><iframe style='width: 1280px; height: 1100px; -ms-zoom: 0.75; -transform: scale(0.68);-moz-transform: scale(0.75);-moz-transform-origin: 0 0; -o-transform: scale(0.75);-o-transform-origin: 0 0; -webkit-transform: scale(0.75);-webkit-transform-origin: 0 0;max-width: none; overflow: hidden; border: 1px solid white;' src='https://lab.montera34.com/contratosmenores/zaragoza/facturas/2017/index_embed.html'></iframe></div>`
